source/server/module.coffee

```diff
@@ -29,15 +31,14 @@ class Space.eventSourcing extends Space.Module
     'Space.eventSourcing.CommitPublisher'
     'Space.eventSourcing.Repository'
     'Space.eventSourcing.Projector'
+    'Space.eventSourcing.Snapshotter'
```

Comment:
This moves down to onInitialize to check if @Configuration.eventSourcing.snapshotter.enabled